src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -321,12 +321,12 @@ private void ValidateMicrosoftAlias(JsonSchema schema, string name, JValue scala
                 {
                     if (_microsoftGraphCache != null)
                     {
-                        var (error, msAlias) = _microsoftGraphCache.GetMicrosoftAliasAsync(alias).GetAwaiter().GetResult();
+                        var (error, msAlias) = _microsoftGraphCache.GetMicrosoftAlias(alias).GetAwaiter().GetResult();
```

Comment:
could we get ride of this GetResult?

Reply:
I'd like to keep it here for now because:
- It is currently the most straightforward structure
- Most of the time (upon cache hit) the returned Task is a completed task, which doesn't involves wait
- There aren't much performance difference now after the change. Retrieving all msalias using @LowEntropyBody 's benchmark without cache is under 7 seconds (similar to bulk call).

Reply:
Will we plan to change it in the future if you say for now? can we add a TODO here?

Reply:
no plan to change it in the future, added a note there

Reply:
I just want to say that we depends on cache too much, @yufeih